### ISSUE TITLE
Add option to lock the backend

### ISSUE THREAD

**Issue by chriskapp** [apioo/fusio#177]
For users who only use the `deploy` command it might be useful to provide a `lock` option for the backend. This means if this lock is set it is not longer possible to create, update or delete resources at the `/backend/*` routes. Users could activate this flag after deployment on a production system so it is no longer possible to change anything. This could prevent user mistakes and it would also increase the security.

**Comment by EvoPulseGaming** [comment]
Is it possible to just create two versions of the backend account? One for whoever deploys the system, and one for "read only" end user? (I think this would be easier, and wouldn't require manually locking as you just wouldn't give them the master account) (Also not sure if I remember correctly but you may have other changes planned that would make this easier)

**Comment by chriskapp** [comment]
Yes with the new scopes for the backend, which are available in the next release, we can create accounts with specific rights i.e. a readonly account. So the idea here was basically to disable the complete backend API but Iam not sure whether this is needed anymore.
